packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -96,14 +152,15 @@ const ReflectionCard = (props: Props) => {
     reactjis,
     reflectionGroupId
   } = reflection
-  const {localPhase, localStage, spotlightGroup, phases, spotlightSearchQuery} = meeting
+  const {localPhase, localStage, spotlightGroup, phases, spotlightSearchQuery, reflectionGroups} =
+    meeting
   const {phaseType} = localPhase
   const {isComplete} = localStage
   const spotlightGroupId = spotlightGroup?.id
   const isSpotlightSource = reflectionGroupId === spotlightGroupId
   const isSpotlightOpen = !!spotlightGroupId
   const atmosphere = useAtmosphere()
-  const reflectionRef = useRef<HTMLDivElement>(null)
+  const reflectionNewRef = useRef<HTMLDivElement>(null)
```

Comment:
I am not sure if `reflectionNewRef` refers to the same thing as `reflectionRef`?

Reply:
I see that you did a `useFragment` refactoring, are you worried about this will affect the `ref`? but here `reflectionNewRef` should not need to be renamed.